sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -840,6 +841,45 @@ class FileBrowserElement extends Polymer.Element {
     }
   }
 
+  private _loadStartupPath() {
+    if (this.fileManagerType === 'jupyter') {
```

Comment:
I think this should be the SettingsManager's responsibility, the file browser shouldn't know where the user settings are stored. So let's keep the old code as is, and just add a TODO that the SettingsManager needs to multiplex?